layouts/_default/list.atom.xml

```diff
@@ -32,14 +32,211 @@
         {{- $copyright = $copyright | replace "&copy;" "©" -}}
         <rights>{{ $copyright | plainify }}</rights>
     {{- end }}
-    {{ with site.Author.name -}}
-        <author>
-            <name>{{ . }}</name>
-            {{ with site.Author.email }}
-                <email>{{ . }}</email>
-            {{ end -}}
-        </author>
-    {{- end }}
+    {{ $authors := "" }}
+    {{ if isset site "Author" }}
+        {{ $authors = site.Author }}
+    {{ else if isset site "Authors" }}
+        {{ $authors = site.Authors }}
+    {{ else if isset site "author" }}
+        {{ $authors = site.author }}
+    {{ else if isset site "authors" }}
+        {{ $authors = site.authors }}
+    {{ else if isset site.Params "Author" }}
+        {{ $authors = site.Params.Author }}
+    {{ else if isset site.Params "Authors" }}
+        {{ $authors = site.Params.Authors }}
+    {{ else if isset site.Params "author" }}
+        {{ $authors = site.Params.author }}
+    {{ else if isset site.Params "authors" }}
+        {{ $authors = site.Params.authors }}
+    {{ end }}
+    {{ if and $authors (reflect.IsSlice $authors) }} <!-- if authors are defined as slice -->
```

Comment:
`if and ..` is not needed when using `with`.

**Please replace all `if`/`isset` with `with`, and the logic will become much cleaner.